deluge/ui/gtkui/gtkui.py

```diff
@@ -212,7 +211,12 @@ def on_die(*args):
         self.ipcinterface = IPCInterface(args)
 
         # Initialize gdk threading
-        gtk.gdk.threads_init()
+        # Gdk Threading
+        # causes crashes in windows
+        # gdk_threads_init has been deprecated since version 3.6
+        # and should not be used in newly-written code.
+        # All GDK and GTK+ calls should be made from the main thread
+        # Gdk.threads_init()
```

Comment:
There are the two commits [64a94ec197ed19a760df3cc0aa8cfbc9b602f2d8] [1f3e930df31060804a119a34f6bb4dc37805ac92] that added threading and I am thinking there is no need for the code anymore.